Created missing functions.

anmaletic opened the pull request.

anmaletic committed:
Created missing functions.

ncaklovic commented:
Testovi prolaze, ali ima nešto za popraviti:
- u programu (`main`) se treba unijeti integer (sa `cin`-a)
- promijenite da `mult_table` prima `std::ostream&` i onda možete iz `main`-a direktno poslati `std::cout`
- ona pogađanja broja razmaka (`digitCount`) možete puno lakše riješiti koristeći [setw](https://en.cppreference.com/w/cpp/io/manip/setw) (iz `<iomanip>`)

anmaletic commented:
Hvala na informaciji, budem promjeni.
Nakon izmjena potrebno je ponovno commitat i napraviti pull request?

anmaletic committed:
Bug fixes.

Added user input in main.
Changed mult_table to accept ostream.
Replaced digitCount with setw in mult_table.

ncaklovic commented:
Sve u redu (jedino ste sad mogli poslati na `cout` direktno `j*m`, ne treba vam string).
